dissect/target/plugins/apps/vpn/openvpn.py

```diff
@@ -61,133 +118,98 @@ class OpenVPNPlugin(Plugin):
     config_globs = [
         # This catches openvpn@, openvpn-client@, and openvpn-server@ systemd configurations
         # Linux
-        "/etc/openvpn/*.conf",
-        "/etc/openvpn/server/*.conf",
-        "/etc/openvpn/client/*.conf",
+        "/etc/openvpn/",
         # Windows
-        "sysvol/Program Files/OpenVPN/config/*.conf",
+        "sysvol/Program Files/OpenVPN/config/",
     ]
 
     user_config_paths = {
-        OperatingSystem.WINDOWS.value: ["OpenVPN/config/*.conf"],
-        OperatingSystem.OSX.value: ["Library/Application Support/OpenVPN Connect/profiles/*.conf"],
+        OperatingSystem.WINDOWS.value: ["OpenVPN/config/"],
+        OperatingSystem.OSX.value: ["Library/Application Support/OpenVPN Connect/profiles/"],
     }
 
     def __init__(self, target) -> None:
         super().__init__(target)
         self.configs: list[fsutil.TargetPath] = []
-        for path in self.config_globs:
-            self.configs.extend(self.target.fs.path().glob(path.lstrip("/")))
+        for base, glob in product(self.config_globs, ["*.conf", "*.ovpn"]):
+            self.configs.extend(self.target.fs.path(base).rglob(glob))
 
         user_paths = self.user_config_paths.get(target.os, [])
-        for path, user_details in itertools.product(user_paths, self.target.user_details.all_with_home()):
-            self.configs.extend(user_details.home_path.glob(path))
+        for path, glob, user_details in itertools.product(
+            user_paths, ["*.conf", "*.ovpn"], self.target.user_details.all_with_home()
+        ):
+            self.configs.extend(user_details.home_path.joinpath(path).rglob(glob))
 
     def check_compatible(self) -> None:
         if not self.configs:
             raise UnsupportedPluginError("No OpenVPN configuration files found")
 
+    def _load_config(self, parser: OpenVPNParser, config_path: fsutil.TargetPath) -> dict:
+        with config_path.open("rt") as file:
+            try:
+                parser.parse_file(file)
+            except ConfigurationParsingError as e:
+                # Couldn't parse file, continue
+                self.target.log.info("An issue occurred during parsing of %s, continuing", config_path)
+                self.target.log.debug(exc_info=e)
```

Comment:
```suggestion
                self.target.log.debug("", exc_info=e)
```
From what I remember, this is necessary.